tests/python-tests.ps1

```diff
@@ -1,5 +1,5 @@
 param (
-    [Version] [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()]
+    [String] [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()]
```

Comment:
In the above script all types are in lower case.

Reply:
Thanks!